### ISSUE TITLE
correct score require browser refresh

### ISSUE THREAD

**Issue by em-mai** [Robooto/LeaderboardAPI#1]
Hi,

first compliments for the system you built!

I noticed the following issue:
after you have played and the game is over you get redirected to the first screen where the score is displayed but it seems the system is always getting the previous score.

So, for example, when you finish the first play in the first screen you don't see the score at all, if you play again in the first screen will get the score for the first play and so on
In other words it seems that in the first screen you always get the score that was recorded for the previous play.
I noticed that you can get the correct score displayed if you hit reload in the browser
Can you replicate this and hopefully fix it or there is something I wrongly implemented?

Thanks
